### DISCUSSION TITLE
TypeError: Subscripted generics cannot be used with class and instance checks

### DISCUSSION THREAD

**Discussion by lizhiqiang778** [initqp/somd#9] [Q&A] [answered]
Hi, thank you for providing such an excellent active learning package. I try to learn and use it.
Recently, I run into this error when I execute the tutorial (H2O.toml, 1.basic directory). If it's convenient for you, please help check this error. Thanks a lot.
![222222](https://github.com/initqp/somd/assets/48582941/3bc7e882-8a8d-42a8-84fb-bae48ae2850c)

**Comment by initqp** [comment, marked as answer]
Hi Zhiqiang Li:

Thanks for trying out SOMD. This is actually a bug in SOMD for python version lower than 3.10, and has been fixed recently in https://github.com/initqp/somd/commit/a8fa041f962ea25ef81c624dea9aa680184172e2. I think you need to install the current `master` version to resolve this.

**Reply by lizhiqiang778** [reply]
ok. I will try it.  Thank you. I can't wait to try this package😀!
